10up/figma-block
# Add playground

### Description of the Change

- This adds blueprint to for enabling live preview on wp.org playground

Closes https://github.com/10up/figma-block/issues/27

### How to test the Change

- Go to https://github.com/10up/figma-block/tree/feature/add-playground?tab=readme-ov-file, confirm you can see the playground label, this is not functional right now, until the plugin is published on wp.org


### Changelog Entry

> Added - Enable playground for Plugin.

### Credits
Props @jeffpaul, @thrijith 

### Checklist:
